src/_pytest/runner.py

```diff
@@ -220,13 +224,21 @@ def call_runtest_hook(item, when: "Literal['setup', 'call', 'teardown']", **kwds
 
 @attr.s(repr=False)
 class CallInfo:
-    """ Result/Exception info a function invocation. """
+    """ Result/Exception info a function invocation.
+
```

Comment:
Looking a bit more, this is missing the most important attribute - the result.

I'd write this:

```
:param result: The return value of the call, if it didn't raise. Can only be accessed if excinfo is None.
:param Optional[ExceptionInfo] excinfo: The captured exception of the call, if it raised.
```

We can't give a type to `result`, at least until (if) `CallInfo` is made generic.

Reply:
Indeed. I took the liberty of updating it myself.

I think if there's not anything else, we can squash/merge this then. 👍